hca/util/__init__.py

```diff
@@ -638,8 +640,8 @@ def build_argparse_subparsers(self, subparsers, help_menu=False):
             docstring = command.__doc__.format(prog=subparsers._prog_prefix)
             method_args = _parse_docstring(docstring)
             command_subparser = subparsers.add_parser(command.__name__.replace("_", "-"),
-                                                      help=method_args['summary'],
-                                                      description=method_args['description'],
+                                                      help=method_data.get("summary"),
```

Comment:
It might be unhelpful to allow methods to not include help and a description.

Reply:
Agreed, please do not make it so that it becomes possible to not include help or a description.

Reply:
you're probably right, but should that cause the CLI and python binding to not work?

Reply:
Yes. If a Swagger API definition is fetched by our client, it's under our control. We should then edit the definition to include help/description.

Reply:
Reversed change